MANUAL.txt

```diff
@@ -6441,6 +6441,35 @@ soft line breaks in Markdown cells; `--atx-headers` will
 cause ATX-style headings to be used; and `--preserve-tabs` will
 prevent tabs from being turned to spaces.
 
+## Image media
+
+When the ipynb input has media such as svg, png, pdf,
+you may want to use `--self-contained` for any of the supported
+output formats including HTML, or use `--extract-media` in general.
+
+## Javascript media
+
+When converting from ipynb, some Jupyter extensions,
+especially those that uses javascript for visualization,
+expects the presence of
+[`require.js`](https://github.com/requirejs/requirejs).
+In this case when converting say from ipynb to html,
+you can add the following
+either via `header-includes` in metadata or
+`--include-in-header` on the command line:
+
+```
+<script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js">
+</script>
+```
+
+such as
+
+```
+pandoc -s -o require.html require.ipynb \
+-V header-includes='<script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js"></script>'
+```
+
```

Comment:
I'm reluctant to add this to the manual, which is already overly long.
Perhaps this information should be in the FAQ on the website?

Reply:
I tried to shorten the description to be more succinct.
I think people are more used to get information like this in the manual than the FAQ,
(where I never went to look for information, but I should now knowing it exists.)

If you insist to make it to FAQ instead,
is it https://pandoc.org/faqs.html ?
Should it be committed to https://github.com/jgm/pandoc-website and which file?

Reply:
FAQs live in this repository, in doc/faqs.md.
They get copied to the website from this repo on each release.

Reply:
Is there a specific error people would get that would tell us that they need require.js?  If so, it would be natural to have a FAQ like "I'm getting such-and-such error when I try to use JavaScript in ipynb. What does this mean?"